implementation/test/DepositFraudTest.js

```diff
@@ -425,8 +524,6 @@ describe("DepositFraud", async function() {
         "Signature is not fraud",
       )
     })
-
-    it.skip("TODO: full test for startSignerFraudLiquidation", async () => {})
```

Comment:
Aligned with the above, it seems like the two TODOs need to be filled in, not replaced with the `start` flows. We should be validating that the completion of the liquidation flows is correct, and if we're in a place where *starting* liquidation insta-liquidates that is… Incorrect? <_<

Reply:
Are you suggesting a full test, starting from deposit cretion and reaching `startSignerFraudLiquidation`?

Reply:
We definitely need one, but not in this PR. This makes more sense to me now after your response to the other comment.